packages/vue/jsx-runtime/dom.d.ts

```diff
@@ -234,7 +234,7 @@ interface AriaAttributes {
 }
 
 // Vue's style normalization supports nested arrays
-export type StyleValue = string | CSSProperties | Array<StyleValue>
+export type StyleValue = undefined | string | CSSProperties | Array<StyleValue>
```

Comment:
I think we can also support all the falsy value, like `null`, `false`

Reply:
Thanks, it makes sense.

But falsy value `0` should be excluded as it is also a number